src/styles/global.scss

```diff
@@ -1,4 +1,5 @@
-// TODO: ADS-129 shuffle imports so we can use space(x) for $border-radius and $box-shadow-primary utility variables
+// TODO: ADS-129 shuffle imports so we can use space(x) for $border-radius-sm and $box-shadow-primary utility variables
```

Comment:
Do we need this comment now that we've installed tokens?

Reply:
Good catch, removed.